src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li id="apple">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li id="samsung">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li id="smartphones">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            id="laptops"
+            data-qa="hover"
+          >
+            <a
+              href="#laptops"
+              class="nav__link"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li id="gadgets">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li id="tables">
+            <a
+              href="#tables"
+              class="nav__link"
+            >
+              Tables
+            </a>
+          </li>
+          <li id="photo">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
```

Comment:
The text 'Photo' should be transformed to uppercase using CSS rather than being typed in uppercase directly. Consider using the `text-transform: uppercase;` CSS property.